test/support/css_selectors.rb

```diff
@@ -74,7 +74,7 @@ def world_locations_selector
   end
 
   def publish_form_selector(document)
```

Comment:
I feel it might be sensible to update the name of this method, given its purpose has changed?